Resources/GameData/kOS/Parts/kOSMachineRad/part.cfg

```diff
@@ -0,0 +1,71 @@
+PART
+{
+// --- general parameters ---
+name = kOSMachineRad
+module = Part
+author = Peter Goddard
+
+// --- asset parameters ---
+mesh = model/model.mu
+scale = 1
+rescaleFactor = 0.99999999999
+iconCenter = 0, 0, 0
+
+// --- node definitions ---
+node_attach = 0.0, 0.0, 0.0, 1, 0, 0
+
+// --- Tech tree ---
+TechRequired = flightControl
+
+// --- editor parameters ---
+cost = 2200
+entryCost = 4200
+category = Control
+subcategory = 0
+title = CompoMax Radial Tubeless
+manufacturer = Squalid-State Devices Inc.
+description = Would you trust life and limb to a mindless autopilot, powered by untested software you hastily wrote yourself? Spacefaring kerbals would!
+
+// attachment rules: stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+attachRules = 0,1,0,0,1
+
+// --- standard part parameters ---
+mass = 0.00001
+dragModelType = default
+maximum_drag = 0.0
+minimum_drag = 0.0
+angularDrag = 0
+crashTolerance = 6
+maxTemp = 3400
+
+
+MODULE
+{
+	name = kOSProcessor
+	diskSpace = 60000

```

Comment:
It’s up for discussion.
I believe it should be treated as a more advanced part.
This is consistent with the gradual progression of technology suggested by the game.
I believe the current parts are at the same level on the tech tree yes?
In the plot laid out for the kOS computer, we are initially emulating 1970’s technology. 
It made sense to me that a more advanced technology (miniaturization, integrated circuits, Nvram etc. ) would get around the  
serious limitations of hard drive space.
Depending on when we expect we might develop global variables, direct data exchange between modules etc. we might want to scale this down.
Another solution might be to add another model for “storage expansion”.

You will also note the weight and drag characteristics of the module.
This is quite consistent with other radial mounted “black boxes” developed for the game.
The problem being we have to mount things on the exterior (with minor exceptions for mods that allow otherwise).\
If drag and weight were significant, that would pose an unrealistic challenge for designing spacecraft, as the engineers would simply locate the 
devices on the interior of the ship.

Additionally, the power consumption is quite high given the hardwired link between the HD size and power draw.
This is only slightly offset by the addition of a weak solar panel.
There are still plenty of challenges to managing this computer through a mission.




> On Mar 18, 2015, at 3:03 PM, Chris Woerz <notifications@github.com> wrote:
> 
> In Resources/GameData/kOS/Parts/kOSMachineRad/part.cfg <https://github.com/KSP-KOS/KOS/pull/679#discussion_r26696115>:
> 
> > +attachRules = 0,1,0,0,1
> > +
> > +// --- standard part parameters ---
> > +mass = 0.00001
> > +dragModelType = default
> > +maximum_drag = 0.0
> > +minimum_drag = 0.0
> > +angularDrag = 0
> > +crashTolerance = 6
> > +maxTemp = 3400
> > +
> > +
> > +MODULE
> > +{
> > +	name = kOSProcessor
> > +	diskSpace = 60000
> 60k is quite a bit bigger than our other processors. It is designed to be an end game part?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/KSP-KOS/KOS/pull/679/files#r26696115>.
>